lib/js_regex/converter/base.rb

```diff
@@ -34,6 +34,9 @@ def apply_quantifier(node)
         if qtf.possessive?
           node.update(quantifier: qtf.text[0..-2])
           return wrap_in_backrefed_lookahead(node)
+        elsif qtf.token == :interval && qtf.text[0..1] == "{,"
+          qtf.text = "{#{qtf.min},#{qtf.max}}"
+          node.update(quantifier: qtf)
```

Comment:
```suggestion
          node.update(quantifier: "{0,#{qtf.max}}")
```

i would do it like this because:

- `quantifier:` also works with Strings, and it is always nice when mutating an object can be avoided
- `0` instead of `qtf.min` makes it a bit more obvious what is happening

Reply:
I did try that at first but the spec spec/lib/js_regex/second_pass_spec.rb:12 fails with 
`NoMethodError: undefined method `min' for "{0,2}":String`
so I figured I needed an object, which is weird because in 35 we do use strings `node.update(quantifier: qtf.text[0..-2])`

Reply:
ah yes, that's because i messed up 😅

using a string here is indeed a problem, and the string passed in line 35 will also cause a `NoMethodError` in a certain rare edge case where second pass needs to re-write quantifiers.

i'll merge this and let you know when i get to fix the other bug and release a new version.

thanks again!

Reply:
awesome, thanks @jaynetics!